generators/server/templates/dotnetcore/src/Project.Domain/Entities/AuditedEntityBase.cs.ejs

```diff
@@ -0,0 +1,25 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+using System;
+using <%= namespace %>.Domain.Interfaces;
+
+namespace <%= namespace %>.Domain {
+    public class AuditedEntityBase : IAuditedEntityBase {
```

Comment:
Also, I don't see a need for the interface IAuditedEntityBase. 
Entities can inherit from this class and they will have the properties avaiable to them and  n save changes check if entity is assignable to AuditedEntityBase instead of the interface